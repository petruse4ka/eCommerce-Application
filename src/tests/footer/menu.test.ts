import { describe, expect, it } from 'vitest';

import Menu from '@/components/footer/menu';

describe('Snapshot test of menu section in footer', () => {
  it('should match the menu section snapshot', () => {
    const menu = new Menu();
    const element = menu.getElement();

    expect(element).toMatchSnapshot();
  });
});
