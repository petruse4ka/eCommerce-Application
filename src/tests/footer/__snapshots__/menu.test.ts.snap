// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer menu > should match the snapshot 1`] = `
<div
  class="flex flex-col items-center lg:items-start gap-6 w-full col-span-1"
>
  <h3
    class="text-xl font-bold text-center md:text-left"
  >
    Полезная информация
  </h3>
  <ul
    class="flex flex-col items-center lg:items-start gap-3 w-full"
  >
    <li
      class="cursor-pointer text-[14px] text-center md:text-left hover:text-pink transition-all duration-300 ease-in-out"
    >
      О компании
    </li>
    <li
      class="cursor-pointer text-[14px] text-center md:text-left hover:text-pink transition-all duration-300 ease-in-out"
    >
      Доставка и оплата
    </li>
    <li
      class="cursor-pointer text-[14px] text-center md:text-left hover:text-pink transition-all duration-300 ease-in-out"
    >
      Условия Конфиденциальности
    </li>
    <li
      class="cursor-pointer text-[14px] text-center md:text-left hover:text-pink transition-all duration-300 ease-in-out"
    >
      Возврат заказа
    </li>
    <li
      class="cursor-pointer text-[14px] text-center md:text-left hover:text-pink transition-all duration-300 ease-in-out"
    >
      Наши контакты
    </li>
  </ul>
</div>
`;
